wagn/lib/wagn/generators/wagn/wagn_generator.rb

```diff
@@ -29,31 +29,29 @@ class WagnGenerator < Rails::Generators::AppBase
 ## should probably eventually use rails-like AppBuilder approach, but this is a first step.  
   def dev_setup
     # TODO: rename or split, gem_path points to the source repo, card and wagn gems are subdirs
-    @gem_path = options['gem-path'] || "ENV['WAGN_DEV_GEM_PATH']"
+    @gemfile_gem_path = @gem_path = options['gem-path']
+    env_gem_path = ENV['WAGN_DEV_GEM_PATH']
+    if env_gem_path.present?
+      @gemfile_gem_path = %q{#{ENV['WAGN_DEV_GEM_PATH']}}
+      @gem_path = env_gem_path
```

Comment:
need to handle these two separately because specs and features need an actual path, whereas it needs to stay a variable in the gemfile.